components/logins/src/db.rs

```diff
@@ -545,7 +571,11 @@ impl LoginDb {
                 }
                 Err(e) => {
                     log::warn!("Skipping login {} as it is invalid ({}).", login.guid, e);
-                    num_failed += 1;
+                    fixup_errors.push(format!(
+                        "Skipping login {} as it is invalid ({}).",
+                        login.guid, e
```

Comment:
Is there any change of `e` containing personally identifiable information? It *shouldn't*, because we try not to log PII and it's being logged directly above, but I'm not sure how carefully we've audited that.

Reply:
I believe we do try hard to avoid PII in errors because we assume they might end up in logcat etc. That said though, the concern is valid and we might not have taken as much care in logins as we did in places - so I'd think this is fine with a followup task to do a quick audit in the logins tree?

Reply:
I'll create an issue for the followup task.

Reply:
Created #2407 to capture this work. I'll tag it for the migration once it makes it's way to JIRA.